OpenKeychain/src/main/res/layout/add_email_dialog.xml

```diff
@@ -21,6 +21,7 @@
         android:layout_marginTop="16dp"
         android:hint="@string/label_email"
         android:imeOptions="actionNext"
+        android:inputType="textEmailAddress"
```

Comment:
not needed due to https://github.com/Hiperzone/open-keychain/blob/development/OpenKeychain/src/main/java/org/sufficientlysecure/keychain/ui/widget/EmailEditText.java#L53